### DISCUSSION TITLE
No terminal is connected

### DISCUSSION THREAD

**Discussion by kranix0** [Taxel/PlexTraktSync#1840] [Support] [answered]
I'm new to docker so I apologise if this is a silly question. I'm trying to run PlexTraktSync on Synology DSM7. Whenever I start the docker instance, it only stays alive for a few seconds. I get these in the logs:

```
Please enter your Plex username or e-mail (): Error
Program requested terminal, No terminal is connected: EOF when reading a line
```

I've tried various ways to attach a terminal but to no avail.

**Comment by glensc** [comment, marked as answer]
who should know what you are doing? your command or anything is not shared here. docker run needs -t for terminal, usually with -it both. what are your various ways? we do not have videos of your screen recordings.
